cotel/src/main/scala/day4/HighEntropyPassphrasses.scala

```diff
@@ -0,0 +1,35 @@
+package day4
+
+import scala.io.Source
+
+object HighEntropyPassphrasses {
+
+  def isValidPassphrase(passphrase: String, checked: List[String] = List(), result: Boolean = true): Boolean = {
+    def recursion(passphrase: List[String], checked: List[String] = List(), result: Boolean = true): Boolean =
```

Comment:
Why not two differents methods? I think because `recursion` method was only used by isValidPassphrase

Reply:
Correct, It's okay to just define an internal function for recursions because they are only going to be used by their wrapper.

In fact, I could just use the parent function as they have the same parameters, but I need to prepare the passphrase a little bit before entering the recursion.